locales/fr_FR/pcsx2_Main.po

```diff
@@ -2852,11 +2843,11 @@ msgstr "Ajustement du Turbo :"
 
 #: pcsx2/gui/Panels/VideoPanel.cpp:78
 msgid "NTSC Framerate:"
-msgstr "Framerate en mode NTSC :"
+msgstr "Taux d'Image en mode NTSC :"
```

Comment:
remove space before :

Reply:
Use "Fréquence des trames" or "d'images".

Reply:
I would use "Rafraichissement" or "IPS" / "Images Par Seconde", as "Taux d'image" or "Fréquence des trames" is not natural in french, too much "Google translate" like.